Cogs/Other.py

```diff
@@ -0,0 +1,289 @@
+            await utility.start_processing(ctx)
+
+            for player in self.helper.get_game_role(game_number).members:
+                thread = await self.helper.get_game_channel(game_number).create_thread(
+                    name=f"ST Thread {player.display_name}",
+                    auto_archive_duration=4320,  # 3 days
+                    type=nextcord.ChannelType.private_thread,
+                    reason=f"Preparing text game {game_number}"
+                )
+
+                await thread.add_user(player)
+                for st in self.helper.get_st_role(game_number).members:
+                    await thread.add_user(st)
+
+            await self.helper.finish_processing(ctx)
+
+            print("-= The CreateThreads command was used successfully by " + str(ctx.author.name) + " at " + str(
+                strftime("%a, %d %b %Y %H:%M:%S ", gmtime()) + "=-"))
+        else:
+            await utility.dm_user(ctx.author, "You are not the current ST for game " + str(game_number))
+            await utility.deny_command(ctx, "CreateThreads")
+
+        await self.helper.log(f"{ctx.author.mention} has run the CreateThreads Command on Game {game_number}")
+
+    @commands.command()
+    async def HelpMe(self, ctx: commands.Context, command_type: typing.Optional[str] = "no-mod"):
+        # Add ShowSignUps here
+        anyone_embed = nextcord.Embed(title="Unofficial Text Game Bot",
+                                      description="Commands that can be executed by anyone", color=0xe100ff)
+        anyone_embed.set_thumbnail(url="https://wiki.bloodontheclocktower.com/images/6/67/Thief_Icon.png")
+
+        anyone_embed.add_field(name=">FindGrimoire",
+                               value="Sends the user a DM listing all games and whether they currently have an ST.",
+                               inline=False)
+        anyone_embed.add_field(name=">ClaimGrimoire [game number]",
+                               value='Grants you the ST role of the given game, unless it is already occupied\n' +
+                                     'Usage examples: `>ClaimGrimoire 1`, `>ClaimGrimoire x3`',
+                               inline=False)
+        anyone_embed.add_field(name=">Enqueue [channel type] [script name] [availability] [notes (optional)]",
+                               value="Adds you to the queue for the given channel type (regular/experimental), "
+                                     "listing the provided information, unless you are already in either of the "
+                                     "queues. Do not join a queue if you are currently storytelling. Note that if a "
+                                     "parameter contains spaces, you have to surround it with quotes.\n" +
+                                     'Usage examples: `>Enqueue regular "Trouble Brewing" "anytime after june"`, '
+                                     '`>Enqueue Exp "Oops All Amnesiacs" "between 07-13 and 07-30" "Let me know '
```

Comment:
American dates >-<

Reply:
yeah 😅 gonna change to something more locale agnostic